scaffold/cmd/templates/source/.github/workflows/test.yaml.tpl

```diff
@@ -22,7 +22,10 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.51.2
+          version: v1.55.2
+          skip-pkg-cache: true
+          skip-build-cache: true
+          args: --timeout=10m
```

Comment:
This is very similar to the change I made locally to workaround https://github.com/cloudquery/cloudquery/issues/15267.